src/server/getters/get-reporting-fees.ts

```diff
@@ -4,22 +4,58 @@ import * as _ from "lodash";
 import { BigNumber } from "bignumber.js";
 import { Address, AsyncCallback } from "../../types";
 import Augur from "augur.js";
+import { QueryBuilder } from "knex";
 import { ZERO } from "../../constants";
 import { groupByAndSum } from "./database";
 
+export interface CrowdsourcerState {
+  address: string;
+  isForked: boolean;
+}
+
+export interface InitialReporterState {
+  address: string;
+  isForked: boolean;
+}
+
+export interface ForkedMarket {
+  address: string;
+  universeAddress: string;
+  isFinalized: boolean;
+  crowdsourcers: Array<CrowdsourcerState>;
+  initialReporter: InitialReporterState | {};
+}
+
+export interface NonforkedMarket {
+  address: string;
+  universeAddress: string;
+  crowdsourcersAreDisavowed: boolean;
+  isFinalized: boolean;
+  isMigrated: boolean;
+  crowdsourcers: Array<CrowdsourcerState>;
+  initialReporterAddress: string | null;
+}
+
 export interface FeeDetails {
   total: {
     unclaimedEth: string;
     unclaimedRepStaked: string;
     unclaimedRepEarned: string;
     lostRep: string;
-    claimedEth: string;
-    claimedRepStaked: string;
-    claimedRepEarned: string;
   };
   feeWindows: Array<Address>;
-  crowdsourcers: Array<Address>;
-  initialReporters: Array<Address>;
+  forkedMarket: ForkedMarket;
```

Comment:
This is not always present right? The type implies it is. In the code, you specify `{} as ForkedMarket` which coerces TypeScript a bit, and allows this incorrect definition (which implies there is always a ForkedMarket here, and a ForkedMarket always has required structure). Something like this would help:


```
  let forkedMarket: ForkedMarket|null = null;
```

Then the type could be :
```
forkedMarket: ForkedMarket|null;
```